diff -crB rtorrent-0.9.6/src/ui/element_download_list.cc rtorrent-0.9.6-hyper_ch/src/ui/element_download_list.cc
*** rtorrent-0.9.6/src/ui/element_download_list.cc	2015-09-03 21:03:30.000000000 +0200
--- rtorrent-0.9.6-hyper_ch/src/ui/element_download_list.cc	2017-10-30 08:55:55.566588987 +0100
***************
*** 65,71 ****
  
    m_bindings['\x13'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "d.start=");
    m_bindings['\x04'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "branch=d.state=,d.stop=,d.erase=");
!   m_bindings['\x0B'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "d.ignore_commands.set=1; d.stop=; d.close=");
    m_bindings['\x12'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "d.complete.set=0; d.check_hash=");
    m_bindings['\x05'] = std::tr1::bind(&ElementDownloadList::receive_command, this,
                                        "f.multicall=,f.set_create_queued=0,f.set_resize_queued=0; print=\"Queued create/resize of files in torrent.\"");
--- 65,71 ----
  
    m_bindings['\x13'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "d.start=");
    m_bindings['\x04'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "branch=d.state=,d.stop=,d.erase=");
!   m_bindings['\x0B'] = std::trl::bind(&ElementDownloadList::receive_command, this, "d.stop=; d.close=");
    m_bindings['\x12'] = std::tr1::bind(&ElementDownloadList::receive_command, this, "d.complete.set=0; d.check_hash=");
    m_bindings['\x05'] = std::tr1::bind(&ElementDownloadList::receive_command, this,
                                        "f.multicall=,f.set_create_queued=0,f.set_resize_queued=0; print=\"Queued create/resize of files in torrent.\"");

